website/scripts/screenshots/src/index.js

```diff
@@ -0,0 +1,180 @@
+const puppeteer = require("puppeteer");
+const {
+  capture,
+  wait,
+  error,
+  click,
+  clickJob,
+  clickTab,
+  clickMainMenu,
+  clickX
+} = require("./utils");
+
+const HOST = process.env.EMBER_HOST || "http://localhost:4200";
+console.log(`Using host ${HOST}...`);
+
+(async () => {
+  const startTime = Date.now();
+  console.log("Preparing puppeteer...");
+
+  // Create a new browser and tab
+  const browser = await puppeteer.launch({
+    // Docker related chrome flags
+    args: [
+      "--no-sandbox",
+      "--disable-setuid-sandbox",
+      "--disable-dev-shm-usage"
+    ]
+  });
+  const page = await browser.newPage();
+
+  // Make sure the page is 4K is high-dpi scaling
+  page.setViewport({ width: 1920, height: 1080, deviceScaleFactor: 2 });
+  console.log("Loading Nomad UI...");
+
+  try {
+    await page.goto(`${HOST}/ui/`);
+  } catch (err) {
+    await error(
+      browser,
+      "Could not load the Nomad UI. Is the Ember server running?"
+    );
+  }
+
+  // Give Mirage a chance to settle
+  console.log("Waiting for Mirage...");
+  await wait(5000);
```

Comment:
I'm sure this is probably fine, but just wondering if there anyway you can check for when Mirage is ready?

Reply:
I think there is some `waitForElement`-like features in puppeteer? My goal was to spike something out quickly rather than well 😄 This is by no means the finest of engineering, but since it's just a script for internal use I figured this sledgehammer approach was adequate.

Reply:
Yep perfect, gotcha 👍